iac-extensions/arm/src/test/resources/checks/ElementsOrderCheckTest/resourceExpected.json

```diff
@@ -0,0 +1,25 @@
+{
+  "resources": [
+    {
+      "comments": "if any",
+      "condition": true,
+      "scope": "% parent scope %",
+      "type": "Microsoft.Compute/virtualMachines",
+      "apiVersion": "2023-09-01",
```

Comment:
With the assumption of case insensitive keys, changing `apiVersion` to `apiversion` would break the test.